frontend/src/pages/Tasks.tsx

```diff
@@ -22,6 +26,47 @@ export default function Tasks() {
     );
   }, [data, error, showSolvedTasks]);
 
+  function App() {
+    const notify = () => {
+      toast.success("🩸First Blood!🩸", {
+        position: "bottom-right",
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: "light",
+        transition: Bounce,
+      });
+    };
+
+    useEffect(() => {
+      if (firstBloodNotification) {
+        notify();
+      }
+    }, [firstBloodNotification]);
```

Comment:
```suggestion
    useEffect(() => {
      if (firstBloodNotification) {
        toast.success("🩸First Blood!🩸", {
          position: "bottom-right",
          autoClose: 5000,
          hideProgressBar: false,
          closeOnClick: true,
          pauseOnHover: true,
          draggable: true,
          progress: undefined,
          theme: "light",
          transition: Bounce,
        });
      }
    }, [firstBloodNotification]);
```

We could either change `notify()` to something more descriptive like `notifyFirstBlood()` or just move into the useEffect as we are going to remove the notify button in the html below